apps/prairielearn/src/schemas/index.ts

```diff
@@ -1,17 +1,17 @@
-// @ts-check
 import * as fs from 'node:fs';
 import * as path from 'node:path';
 
 /**
  * Reads a JSON file from the schemas directory.
- * @param {string} filePath
+ * @param filePath
  */
-function readSchema(filePath) {
+function readSchema(filePath: string) {
   return JSON.parse(
     fs.readFileSync(path.resolve(import.meta.dirname, 'schemas', filePath), 'utf8'),
   );
 }
 
+// TODO: Add TypeScript types for each schema automatically
```

Comment:
Nit, I think we can drop this comment since we're changing strategies as discussed in #11014.

```suggestion
```